ext/bg/js/settings.js

```diff
@@ -93,13 +104,37 @@ function formUpdateVisibility(options) {
     }
 }
 
+async function formMainDictionaryOptionsPopulate(options) {
+    const select = $('#main-dictionary').empty();
+
+    let titles = await utilDatabaseGetTitlesWithSequences();
+    titles = titles.filter(title => options.dictionaries[title].enabled);
+    const formOptionsHtml = [];
+    let mainDictionarySelected = false;
+    for (title of titles) {
```

Comment:
You probably mean `for (const title of titles)`; not having `const` here makes it this look kind of wonky.